common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -631,4 +657,13 @@ private static void setCorrelationIdOnResult(@NonNull final CommandResult comman
         }
     }
 
+
+    private static void performRefresh(BaseCommand command) {
+        SilentTokenCommandParameters parameters = ((SilentTokenCommandParameters) command.getParameters()).toBuilder().forceRefresh(true).build();
```

Comment:
As I read this, this will be active on interactive requests as well.  If that's true, you'll get a ClassCastException here.  Is it?

Reply:
Offline: I missed where this is actually triggered, so it won't be a concern for now.  Think we should check and log an error (and return) before we cast the value, though, just to prevent anybody from accidentally tripping on it.

Reply:
We should probably modify the input to accept only `SilentTokenCommand` objects